Makefile

```diff
@@ -1,41 +1,14 @@
-VERSION = 0.34
-all: flexlink.exe support
-
-include $(shell cygpath -ad "$(shell ocamlopt -where)/Makefile.config")
-
-MINGW_PREFIX = i686-w64-mingw32
-MINCC = $(MINGW_PREFIX)-gcc
-
-MINGW64_PREFIX = x86_64-w64-mingw32
-MIN64CC = $(MINGW64_PREFIX)-gcc
-
-CYGWIN_PREFIX = i686-pc-cygwin
-CYGCC = $(CYGWIN_PREFIX)-gcc
-
-CYGWIN64_PREFIX = x86_64-pc-cygwin
-CYG64CC = $(CYGWIN64_PREFIX)-gcc
-
-.PHONY: version.ml
-version.ml:
-	echo "let version = \"$(VERSION)\"" > version.ml
-	echo "let mingw_prefix = \"$(MINGW_PREFIX)\"" >> version.ml
-	echo "let mingw64_prefix = \"$(MINGW64_PREFIX)\"" >> version.ml
-
-# Supported tool-chains
-
-CHAINS = mingw mingw64 cygwin cygwin64 msvc msvc64
-
-# Compilers
+ifeq ($(findstring clean,$(MAKECMDGOALS)),)
+include Makefile.config
+endif
 
-# Attempt to locate the Windows SDK
+all: $(if $(filter-out no,$(HAVE_OCAMLOPT)),flexlink.exe) support
 
-ifeq ($(findstring clean,$(MAKECMDGOALS)),)
-include Makefile.winsdk
+ifneq ($(filter-out no,$(HAVE_OCAMLOPT)),)
```

Comment:
Eliminates the warning about `Makefile.config` not being found (and ocamlopt not being found) when bootstrapping.